gradle/detect.go

```diff
@@ -95,7 +96,44 @@ func (Detect) Detect(context libcnb.DetectContext) (libcnb.DetectResult, error)
 					},
 				},
 			},
-		}, nil
+		}
```

Comment:
```suggestion
		}
		break
```

The previous behavior here would break out after it finds the first file, because it would return. I'm not sure this is a huge deal, but you can probably break after you find the first file.

Reply:
I noticed this but left out adding it in to avoid changing logic that doesn't relate to this PR. If you don't feel too strongly I think we should fix this separately?